doc/modules/ROOT/pages/index.adoc

```diff
@@ -1,46 +1,577 @@
+* [x] Able to establish SSH connections to instances
+====
 
-include::{templates-url}/link-to-repo.adoc[]
+== 1) Create and Configure a VPC
 
-== What You’ll Learn
+Let's start with creating a VPC on which our instances will run.
+
+=== 1.1) Create a VPC
+
+[tabs]
+====
+
+{tab-header-console}::
++
+--
+Under `VPC > Your VPCs` section of AWS console, create a new VPC and give it a name you can remember in the next steps:
+
+image::ss1-create-vpc.png[]
+--
+
+{tab-header-CLI}::
++
+--
+Create a VPC with a cidr block and note the *VpcId* to use afterwards:
+
+[source, shell]
+----
+$ aws ec2 create-vpc --cidr-block 10.0.0.0/16
+
+Vpc:
+  CidrBlock: 10.0.0.0/16
+  CidrBlockAssociationSet:
+  - AssociationId: vpc-cidr-assoc-***********
+    CidrBlock: 10.0.0.0/16
+    CidrBlockState:
+      State: associated
+  DhcpOptionsId: dopt-*******
+  InstanceTenancy: default
+  Ipv6CidrBlockAssociationSet: []
+  IsDefault: false
+  OwnerId: '***********'
+  State: pending
+  VpcId: vpc-***********
+----
+
+Then add a name tag you can remember in the next steps:
+[source, shell]
+----
+$ aws ec2 create-tags --resources vpc-*********** \
+    --tags Key=Name,Value=hazelcast-guide
+----
+--
+====
+
+==== 1.1.1) Create an Internet Gateway
+
+Now create an internet gateway to be attached to your VPC. You will need this gateway to establish SSH connections to
+instances:
+
+[tabs]
+====
+
+{tab-header-console}::
++
+--
+image::ss1.1.1-create-igw.png[]
+--
+
+{tab-header-CLI}::
++
+--
+
+Note the *InternetGatewayId* when it is created:
+
+[source, shell]
+----
+$ aws ec2 create-internet-gateway
+
+InternetGateway:
+  Attachments: []
+  InternetGatewayId: igw-**************
+  OwnerId: '**************'
+  Tags: []
+----
+
+Then add a name tag to make it distinguishable:
+[source, shell]
+----
+$ aws ec2 create-tags --resources igw-************** \
+    --tags Key=Name,Value=hazelcast-guide
+----
+--
+====
+
+==== 1.1.2) Attach the Internet Gateway
+
+After creating an internet gateway, attach it to your VPC.
+
+[tabs]
+====
+
+{tab-header-console}::
++
+--
+Navigate to `VPC > Internet gateways > Attach to VPC` section. Choose the VPC you just created among the available ones:
+
+image::ss1.1.2-attach-igw.png[]
+--
+
+{tab-header-CLI}::
++
+--
+
+Using their id's, attach the gateway to the VPC:
+
+[source, shell]
+----
+$ aws ec2 attach-internet-gateway \
+    --internet-gateway-id igw-*********** \
+    --vpc-id vpc-***********
+----
+--
+====
+
+=== 1.2) Add Default Route
+
+Now add the default route to the route table. This step will make you able to connect to instances via SSH.
+
+[tabs]
+====
+
+{tab-header-console}::
++
+--
+Under the VPC description, navigate to route table shown in the orange box:
+
+image::ss1.2.1-nav-to-routes.png[]
+
+Under `Routes` tab, you need the default one (`0.0.0.0/0`) listed. `Edit routes` to add this one:
+
+image::ss1.2.2-list-routes.png[]
+
+As the target, pick the internet gateway you created in section `1.1.1` and save:
```

Comment:
I think, in-page section reference would be better for better user-experience:
https://docs.antora.org/antora/2.3/asciidoc/in-page-xref/